lib/blog-feed-loader.js

```diff
@@ -1,2 +1,6 @@
-// TODO: Replace this with a real implementation.
-module.exports = require('../test/browser/stub-blog-feed-loader');
+if (process.env.NODE_ENV !== 'production' &&
```

Comment:
This check ensures that our stub loader is never included in the production webpack bundle ([explanation](https://github.com/webpack/docs/wiki/list-of-plugins#defineplugin)).